__tests__/cdk_postMapper/new_relic.test.js

```diff
@@ -0,0 +1,131 @@
+const { commonPostMapper } = require("../../cdk/new_relic/transform");
+
+describe("Unit test cases for new_relic common post mapper", () => {
+  let payload, event, rudderContext;
+  beforeEach(() => {
+    payload = { event: "first", timestamp: 1580602989, abc: "123", key: 124 };
+    event = {
+      message: { userId: "identified user id", anonymousId: "anon-id-new" },
+      destination: {
+        Config: {
+          accountId: "12345",
+          insertKey: "11111122702j2a2U2K2C7H",
+          dataCenter: "us"
+        }
+      }
+    };
+    rudderContext = {};
+  });
+
+  it('If user did not provid a eventType name, then we will include "rudderstack" in the payload directly', () => {
+    event = {
```

Comment:
Rather than re-populating the whole event object, we can probably just manipulate the delta 
Something like this.
``` javascript
event.destination.Config.customEventType: "";
event.destination.Config.sendDeviceContext = false
```